tests/scripts/quiet/quiet

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env bash
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+#
+# This swallows the output of the wrapped tool, unless there is an error.
+# This helps reduce excess logging in the CI.
+
+# If you are debugging a build / CI issue, you can get complete unsilenced logs
+# by un-commenting the following line (or setting VERBOSE_LOGS in your environment):
+# VERBOSE_LOGS=1
+
+# Locate original tool
+TOOL_WITH_PATH=$(dirname "$0")/$TOOL
+ORIGINAL_TOOL=$(type -ap "${TOOL}" | grep -v -Fx "$TOOL_WITH_PATH" | head -n1)
+
+print_quoted_args() {
+    # similar to printf '%q' "$@"
+    # but produce more human-readable results for common/simple cases like "a b"
+    for a in "$@"; do
+        simple_pattern='^([[:alnum:]_+-]+=)?([[:alnum:] _=+-./:@]*)$'
+        if [[ $a =~ ' ' && $a =~ $simple_pattern ]]; then
```

Comment:
Why not use single quotes? Then the only character you'd have to worry about is `'` itself.
```suggestion
        if [[ $a =~ ' ' && $a =~ ^([-+./:_[:alnum:]]=)?[^\']+$ ]]; then
```

Also, how about triggering this path with other characters that need quoting, e.g. for `MBEDTLS_CONFIG_FILE="my/config.h"`?
```suggestion
        if [[ $a =~ [^-+.,/:=@[:alnum:]_] && $a =~ ^([^-+.,/:@[:alnum:]_]=)?[^\']+$ ]]; then
```